### ISSUE TITLE
Camera Orientation View

### ISSUE THREAD

**Issue by avinashgpai** [google/model-viewer#2063]
### Description

Hi all,

Currently model and animation are viewed from default camera provided by model-viewer( as per my knowledge). Is there any way one can view through camera that's exported along with model.

Below I have attached a drive link where there is video in which you can see I have used https://gltf-viewer.donmccurdy.com/ to view model. In camera there is 2 options default and camera_orientation. I can relate model-viewer camera to default camera from the above site and I wanted to know is there any way to view from camera_orientation using model-viewer? If not I would like to be this as a feature.  

Thanks,
Avinash

**Comment by elalish** [comment]
Indeed, this has been on the back-burner for some time. Probably not that hard to implement; interested in contributing?

**Comment by avinashgpai** [comment]
I would really love to but I don't have enough knowledge and don't wanna mess the existed code😁. I would love to see this to be feature soon 😊.

**Comment by avinashgpai** [comment]
Any update on this feature?

**Comment by jontojek** [comment]
Yes please, would very much like to load and view through a user defined camera.

**Comment by elalish** [comment]
@jontojek user-defined is no problem: that's what `camera-target` and `camera-orbit` are for. You can even not put `camera-controls` in so there's no default interaction. Is that adequate, or are you really looking for a glTF-defined camera?

**Comment by jontojek** [comment]
Thank you for the response Elaish, and yes I have been using Yaw and Pitch in the Model Editor GUI.  
camera-orbit="145deg 90deg 33.57m" field-of-view="30deg
So those two variables seem to work well, but field-of-view does not take any effect.  I have a reference photograph that I am trying to match with ModelViewer, so Yaw and Pitch are possible to match.  The photo has field-of-view = 90, and I would like to get ModelViewer to replicate that, but unfortunately it seems that the ModelViewer camera is locked to fov=45.

**Comment by elalish** [comment]
You simply need to set the `max-field-of-view` large enough to not clamp.

**Comment by jontojek** [comment]
Thank you elalish, by adjusting max-field-of-view, I can now get fov to change.

**Comment by tishok** [comment]
Any news?

**Comment by elalish** [comment]
PRs welcome!

**Comment by eyaler** [comment]
> I added this code to ModelScene class:
> 
> ```
> /* ..... */
> if (gltf != null) {
>     gltf.scene.traverse(
>         object => {
>             if (object.isCamera) {
>                 this.camera = object;
>             }
>         }
>     );
> 
>     this._model = gltf.scene;
>     this.target.add(gltf.scene);
> }
> /* ..... */
> ```
> 
> It works, but i think you could make it better.

thanks @tishok this gives me the correct camera angles and position, however i can no longer move the camera. if i just want to use the blender camera for the initial position and angles - how should i change this?
